Mu2eG4/src/Mu2eGammaDaughterProcess.cc

```diff
@@ -0,0 +1,99 @@
+// Special process to kill events with low energy photon daughters
+//
+// Original author M. MacKenzie
+
+// Mu2e includes
+#include "Mu2eG4/inc/Mu2eGammaDaughterProcess.hh"
+#include "Mu2eG4/inc/Mu2eG4UserHelpers.hh"
+
+// G4 includes
+#include "G4ios.hh"
+#include "G4VParticleChange.hh"
+#include "G4Track.hh"
+#include "G4Step.hh"
+
+namespace mu2e{
+
+  Mu2eGammaDaughterProcess::Mu2eGammaDaughterProcess(const G4String& aName)
+    : G4VDiscreteProcess(aName,fUserDefined)
+  {
+    verbose_ = 10;
+    accepted_ = 0;
+    photonEnergy_ = -1.;
+    theProcessSubType = 0;
```

Comment:
I see that Mu2eRecorderProcess.cc also defines theProcessSubType=0.  Is there a risk of conflict or ambiguity here?  Maybe Krzysztof should comment.

Reply:
I'm not sure what `theProcessSubType` does, so I'll let Krzysztof respond to this.

Reply:
I do not see it being used for user fUserDefined type processes
I suggest we worry about it when we find a use case for it